code/modules/power/supermatter/supermatter_process.dm

```diff
@@ -266,11 +280,10 @@
 		playsound(src, 'sound/weapons/emitter2.ogg', 70, TRUE)
 		var/power_multiplier = max(0, (1 + (power_transmission_bonus / (10 - (gas_comp[/datum/gas/bz] * BZ_RADIOACTIVITY_MODIFIER)))) * freonbonus)// RadModBZ(500%)
 		var/pressure_multiplier = max((1 / ((env.return_pressure() ** pressure_bonus_curve_angle) + 1) * pressure_bonus_derived_steepness) + pressure_bonus_derived_constant, 1)
-		var/co2_power_increase = max(gas_comp[/datum/gas/carbon_dioxide] * 2, 1)
```

Comment:
you could have this be `max(1 + gas_comp[/datum/gas/carbon_dioxide] * 0.5, 1)` 
this way it would be a multiplier from 1 to a theoretical maximum of 1.5

Reply:
Would there be a reason to keep the max check, since that would always be at least 1 assuming the composition wouldn't be negative?